src/core/contracts/identity.ts

```diff
@@ -100,8 +100,8 @@ export const createBeaconProxyWithOwner = async (
 };
 
 /**
- * Create a new identity and register it to a handle to get a new DSNP User Id.
- * This will create and register a new beacon proxy identity.
+ * createAndRegisterBeaconProxy() Create a new identity and register it to a
+ * handle to get a new DSNP Id. This will create and register a new beacon proxy identity.
```

Comment:
This line is over 80 characters. I'm not sure if we've settled on it as a rule yet, but I try to wrap my doc comments at 80. In any case, we need to decide on a convention for consistency.